clients/js/src/embeddings/TransformersEmbeddingFunction.ts

```diff
@@ -30,7 +31,7 @@ export class TransformersEmbeddingFunction implements IEmbeddingFunction {
       // Also, since we use `"module": "commonjs"` in tsconfig.json, we use the following workaround to ensure
```

Comment:
We can remove this comment. I have added a PR which resolves the problem that some frameworks import the cjs build instead of the esm version: https://github.com/chroma-core/chroma/pull/963